src/ui/components/Events/Events.tsx

```diff
@@ -30,20 +45,18 @@ function Events({ currentTime, events, executionPoint, seek }: PropsFromRedux) {
   );
 
   return (
-
-    <div className="flex flex-col py-1.5 self-stretch space-y-1.5 w-full text-xs bg-white">
-    
+    <div className="flex flex-col py-1.5 self-stretch w-full text-xs bg-white">
       {events.map((e, i) => {
         return (
           <div key={e.point}>
-            {i === currentEventIndex ? <CurrentTimeLine /> : null}
+            <CurrentTimeLine currentEventIndex={currentEventIndex} index={i} />
```

Comment:
I could go either way on this, but it seems like maybe a lot to pass around these two variables just to do an equality check on them later. What do you think about having an `active` state, and this component can render `<CurrentTimeLine active={i === currentEventIndex}/>`?

Reply:
👍